pkgs/misc/emulators/mednaffe/default.nix

```diff
@@ -0,0 +1,28 @@
+{ stdenv, fetchFromGitHub, pkgconfig, gtk2, mednafen }:
+
+stdenv.mkDerivation rec {
+
+  version = "0.8";
+  name = "mednaffe-${version}";
+
+  src = fetchFromGitHub {
+	repo = "mednaffe";
+	owner = "AmatCoder";
+	rev = "v${version}";
+	sha256 = "1j4py4ih14fa6dv0hka03rs4mq19ir83qkbxsz3695a4phmip0jr";
+  };
+
+  prePatch = ''
+    substituteInPlace src/mednaffe.c --replace "binpath = NULL" "binpath = \"${mednafen}/bin/mednafen\""
+  '';
+
+  buildInputs = with stdenv.lib; [ pkgconfig gtk2 mednafen ];
+
+  meta = with stdenv.lib; {
+    description = "A GTK based GUI frontend for mednafen, portable, CLI-driven, SDL+OpenGL-based, multi-system emulator";
```

Comment:
This description is way to long, in my opinion. Stating that it's a gui front-end for mednafen suffices, I think.